src/components/FeedbackPage/FeedbackPage.js

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import * as styles from '../../styles/FeedbackPage.module.css';
+
+function FeedbackPage(props) {
```

Comment:
your feedback page supposed to save to local storage

Reply:
Yes, I've made it so that when the user clicks the Submit button, the data is saved to local storage, I'm just trying to make redux work before pushing the code.

Reply:
ah, I thought you're ready with it. My bad/